Source/WebCore/html/HTMLElement.cpp

```diff
@@ -834,7 +835,7 @@ std::optional<SRGBA<uint8_t>> HTMLElement::parseLegacyColorValue(StringView stri
     if (equalLettersIgnoringASCIICase(string, "transparent"_s))
         return std::nullopt;
 
-    if (auto namedColor = CSSParser::parseNamedColor(string))
+    if (auto namedColor = CSSParserFastPaths::parseNamedColor(string))
```

Comment:
Seems a little bit of a naming mistake to have some _entry points_ into the parser be in a namespace `CSSParserFastPaths`. With a name like that I’d expect it to be an implementation detail of the parser, hidden behind a light abstraction.

Reply:
I agree. I am going to move all these color parsing functions in a follow up.